plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3/TestTrinoS3FileSystem.java

```diff
@@ -337,13 +338,15 @@ public void testReadNotFound()
             fs.initialize(new URI("s3n://test-bucket/"), new Configuration(false));
             fs.setS3Client(s3);
             try (FSDataInputStream inputStream = fs.open(new Path("s3n://test-bucket/test"))) {
-                inputStream.read();
+                assertThatThrownBy(() -> inputStream.read())
+                        .isInstanceOf(IOException.class)
+                        .hasMessageMatching(".*Failing getObject call with " + HTTP_NOT_FOUND + ".*");
```

Comment:
You could do `hasMessageContaining` and drop the `.*` on both ends

Reply:
Thanks for the feedback. I will go ahead and make these changes.